### DISCUSSION TITLE
Redownload movies deleted by real debrid

### DISCUSSION THREAD

**Discussion by IIIEII** [itsToggle/plex_debrid#476] [Q&A]
Hello!
Recently I forgot to renew my real debrid subscription and all my downloaded movies got deleted. In plex now I see red Trash Can icon on every movie in debrid library. What is the proper way to download them again after upgrading subscription? Should I add them all to watchlist manually or something like this? Or maybe there is some option in plex_debrid to trigger redownload?

**Comment by arjanterheegde** [comment]
I'm using Overseerr for all my movie or serie requests. If my Real-Debrid downloads are deleted i can update my whole Overseerr database so Plex Debrid thinks it's not requested yet and will scrap it again.

**Comment by townsmcp** [comment]
> I'm using Overseerr for all my movie or serie requests. If my Real-Debrid downloads are deleted i can update my whole Overseerr database so Plex Debrid thinks it's not requested yet and will scrap it again.

@arjanterheegde what do you do to Overseerr to make that happen?

**Reply by arjanterheegde** [reply]
I've opened the Overseerr database and set all requests to "requested" again. Don't know the exact query anymore, should have saved it.
